t/base/rs.t

```diff
@@ -1,312 +1,455 @@
 #!./perl
+
+# t/base/rs.t - $/ variable testing
+#
+# Test plan
+#
+# $/ allowed values:
+# ------------------
+#
+#   By default  - "\n"
+#   "\n"        - explicit value
+#   3           - non line terminator
+#   34          - more than one symbol
+#   ''          - delimiter as a paragraph
+#   undef       - the whole file
+#   2           - integer
+#   "2"         - string delimiter
+#
+#   \2          - part symbols count of
+#   \"2"        - a delimited string
+#
+# $/ unallowed values:
+# --------------------
+#
+#   \0       - Setting $/ to a reference to zero is forbidden
+#   \-1      - Setting $/ to a reference to a negative integer is forbidden
+#   []       - Setting $/ to an ARRAY reference is forbidden
+#   {}       - Setting $/ to a HASH reference is forbidden
+#   \\1      - Setting $/ to a REF reference is forbidden
+#   qr/foo/  - Setting $/ to a REGEXP reference is forbidden
+#   \*STDOUT - Setting $/ to a GLOB reference is forbidden
+#
+# Test preserverd value after trying to set unallowed ones
+# Tests for VMS case
+# Test $/ by reading file to "my" filehandle variable
+# Test $/ by reading file to "our" filehandle variable
+# Test $/ for files from variables like real files
+
 # Test $/
 
 print "1..41\n";
 
+# ======================================================
+# Preparation
+# ======================================================
+
 $test_count = 1;
 $teststring = "1\n12\n123\n1234\n1234\n12345\n\n123456\n1234567\n";
 $teststring2 = "1234567890123456789012345678901234567890";
 
-# Create our test datafile
-1 while unlink 'foo';                # in case junk left around
+# ------------------------------------------------------
+# Remove previous test files if exist
+# ------------------------------------------------------
+# In case of existing files with the same 'foo' name
+# on some systems with versioning file system (e.g VMS).
+# Read more at pod/perport.pod "System Interaction"
```

Comment:
typo in filename... and I'm not sure if the explanation in this comment is correct.

Reply:
The typo, yes, thank you. What explanation should be used here instead?